src/ui/inspector/ElementsPool.vue

```diff
@@ -34,7 +34,7 @@
         <ul
             v-if="elements.length > 0"
             id="inspector-elements-tree"
-            class="c-tree c-elements-pool__tree"
+            class="c-tree c-elements-pool__tree js-elements-pool__tree"
```

Comment:
we can select on the `id` here instead of adding another class for e2e

Reply:
I'm following the pattern you used for Overlay Plots where you used the js class. Has that pattern changed?